Fix example reference in git function tutorial (#1382)
kyma-project/serverless: examples/orders-service/README.md

# Overview

> A serverless Function with the ability to expose HTTP endpoints used to read all order records or post single orders. Function can run with either the default in-memory database or the external Redis instance. 

Reviewer: ```suggestion
This example provides a Serverless Function that exposes HTTP endpoints to read all order records or post single orders. The Function can run with either the default in-memory database or the external Redis instance. 
```

## Prerequisites

> - Serverless module installed locally or on a cluster

Reviewer: ```suggestion
- [Serverless module installed](https://kyma-project.io/#/02-get-started/01-quick-install) locally or on a cluster
```

## Deploy Function

> 1. To deploy this function, follow the tutorial presented [here](https://kyma-project.io/#/serverless-manager/user/tutorials/01-11-create-git-function) 

Reviewer: ```suggestion
1. To deploy this Function, follow the [Create a Git Function](https://kyma-project.io/#/serverless-manager/user/tutorials/01-11-create-git-function) tutorial.
```
